pcsx2/SaveState.cpp

```diff
@@ -233,6 +233,14 @@ SaveStateBase& SaveStateBase::FreezeInternals()
 	// to merit an HLE Bios sub-section... yet.
 	deci2Freeze();
 
+#ifndef DISABLE_RECORDING
```

Comment:
Did a local test and the define will have to be removed here. If the user were to make a savestate without the tools enabled and then load it with them enabled; an error will be thrown because the tag wasn't there during creation.

It was non-fatal in my case but we don't want to take chances. I'd guess this is a bit safer anyways.
```
Savestate is corrupt or incomplete!
Path: [Unnamed or unknown]
(pxActionEvent) Savestate data corruption detected while reading tag: InputRecording(VM_UnzipFromDisk)
```